src/Data/Cursor/AbstractCursor.php

```diff
@@ -134,6 +144,99 @@ protected function is_valid_offset_and_node() {
 		return true;
 	}
 
+	/**
+	 * Validates threshold field configuration. Validation failure results in a fatal
+	 * error because query execution is guaranteed to fail.
+	 *
+	 * @param array $field  Threshold configuration.
+	 * 
+	 * @throws InvariationViolation Invalid configuration format.
+	 * 
+	 * @return void
+	 */
+	protected function is_valid_threshold_field( $field ) {
+		$class = get_called_class();
+		// Throw if an array not provided.
+		if ( ! is_array( $field ) ) {
+			$type  = gettype( $field );
+			throw new InvariantViolation( __( "Invalid value provided for {$class} threshold field. Expected Array, ${type} given.", 'ql-events' ) );
```

Comment:
(here and below), your translation namespace is leaking. Should be `wp-graphql`

Reply:
🤷🏿‍♂️